src/services/config.js

```diff
@@ -150,12 +160,21 @@ class ConfigData {
     configObj.projects = configObj.projects || [];
     configObj.projects.forEach((project) => this.addProfile(project.id, project.accountSid, project.region));
     this.setActiveProfile(configObj.activeProject);
+    this.profiles = configObj.profiles || {};
   }
 
   sanitize(string) {
     // Trim whitespace if given a non-null string.
     return string ? string.trim() : string;
   }
+
+  addApiKeysToProject(profile) {
```

Comment:
I don't see the need for this method. If the profile was pulled from `profiles` already, it should have the API info.